environment/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.14.2-buster as build
+FROM ubuntu:18.04
```

Comment:
This change is wrong, ethhashproof need golang to build. Also there must be `as` since dockerfile cannot have two plain `FROM`

Reply:
That is my mistake but not sure even if we need this anymore. I was thinking about deleting this file all together because we don't have the ethhashproof command implemented yet, it was implemented in the python version if I recall correctly. Currently this is not used. So maybe we should delete it and add the golang once we have the nodejs implementation done. cc: @nearmax your opinion?

Reply:
I think this file can be deleted, however our major dockerfile need to strip more "development dependencies" as a production docker image, like what we did in this dockerfile:
```
FROM golang:... as build-ethhashproof
# build ethhashproof
FROM rust:... as build-rust-contracts
# build a few wasm files
FROM node:14
# The entry point nodejs app
# build sol contracts 
# copy built ethhashproof binary into image
# copy built rust contracts into image
```

Reply:
Deleted the file.